lib/ReactViews/Feedback/FeedbackForm.jsx

```diff
@@ -3,6 +3,7 @@
 import ObserveModelMixin from '../ObserveModelMixin';
 import React from 'react';
 import sendFeedback from '../../Models/sendFeedback.js';
+import shareLink from '..//Map/Panels/SharePanel/BuildShareLink.js';
```

Comment:
Extra slash, and also I think it's good practice to make the name of the import match the name of the file.  It's a lot less confusing that way.